README.md

```diff
@@ -3,5 +3,13 @@
 In this project, I will set up a "Hello world" repository and message. My goal here is to master all of the tools and best practices that I learned in Microverse. 
 I hope It demostrates my profesionalism and my capability to document any project in a rigth way!
 
+Requirements:
+- It has files like .html, and .css to show the message "Hello Microverse!"
+- It has linters that works to detect some mistakes
+- It has all the requirements asked for Microverse for this project
+- It has a .gitignore file
+- It has a README.md file to describe this project rigthly
+
+
 Thanks,
 Oscar De la Cruz
```

Comment:
- [ ] A README file is used to communicate important information about your project and also contains all the necessary information for developers to get started using and contributing to your project.

- [x] Please kindly note that all main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in your repo to include all important information about your project 😉.

![image](https://user-images.githubusercontent.com/78251927/211880374-b1f94fd5-9670-4268-8580-4d033a6e14b2.png)

- [ ] The only optional section is the FAQ. Please kindly update your readme and customize all entries accordingly 👍